src/SimpleFeatureToggle/FileFeatureLoadingStrategy.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System.Diagnostics;
+
+namespace SimpleFeatureToggle
+{
+    public class FileFeatureLoadingStrategy : IFeatureLoadingStrategy
+    {
+        private readonly IFileReader _fileReader;
+        private readonly string _fileName;
+
+        public FileFeatureLoadingStrategy(string fileName, IFileReader fileReader = null)
```

Comment:
I was just thinking it'll avoid people thinking that they might have to implement `IFileReader`. We are only really using the `IFileReader` so we can mock it in our tests :wink:

I used it [here](https://github.com/mrstebo/SimpleFeatureToggle/pull/2/files#diff-b52c746df9c242fac3678bf30a8b7186L19)